projects/rocprofiler-systems/docker/Dockerfile.rhel.ci

```
@@ -20,7 +20,8 @@ ARG NJOBS="8"
 RUN yum groupinstall -y "Development Tools" && \
     yum install -y epel-release && crb enable && \
     yum install -y --allowerasing chrpath cmake curl dpkg-devel iproute ninja-build numactl-devel \
-        openmpi-devel papi-devel python3-devel python3-pip sqlite-devel texinfo wget which vim zlib-devel && \
+        openmpi-devel papi-devel python3-pip sqlite-devel texinfo wget which vim zlib-devel \
+        json-devel && \
```

```suggestion
    yum install -y --allowerasing chrpath cmake curl dpkg-devel iproute json-devel \
        ninja-build numactl-devel openmpi-devel papi-devel python3-pip sqlite-devel \
        texinfo vim wget which zlib-devel && \
```
Sorting list alphabetically.